libs/s25main/ingameWindows/IngameWindows.h

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2005 - 2020 Settlers Freaks (sf-team at siedler25.org)
+//
+// This file is part of Return To The Roots.
+//
+// Return To The Roots is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// Return To The Roots is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+#include "iwAIDebug.h"
```

Comment:
if you only need these in the cpp, include it there, otherwise you include it everywhere you include this file

Reply:
After consideration, I removed the IngameWindows.h header altogether. The includes are moved back to dskGameInterface.cpp and the map of persistent windows I moved to Settings.cpp.